package.json

```diff
@@ -17,13 +17,15 @@
     "yargs": "^12.0.2"
   },
   "devDependencies": {
-    "@types/yargs": "^13.0.3",
     "@types/inquirer": "^6.5.0",
     "@types/jest": "^24.0.20",
+    "@types/sinon": "^7.5.0",
     "@types/node": "^12.11.7",
     "@types/pino": "^5.8.13",
+    "@types/yargs": "^13.0.3",
     "aws-sdk-mock": "^4.5.0",
     "jest": "^24.9.0",
+    "sinon": "^7.5.0",
```

Comment:
Are we using Sinon for a specific reason?
If not, I think it will be better to stick to jest for spying, mocking feature instead of using Sinon to keep things consistent as we are using Jest for these purposes in all of our Lendi  TS/JS projects.

Reply:
Yep. The reason is to spy on `aws-sdk` functions. I saw that you could use sinon spies in conjunction with `aws-sdk-mock`. (For reference, see https://github.com/dwyl/aws-sdk-mock#sinon)

Reply:
And since I used sinon for this use case, I ended updating all the other spies that used `Jest` to `sinon` for consistency